src/components/Book/CategoryView.tsx

```diff
@@ -7,16 +9,28 @@ type Props = {
 };
 export function BookCategoryView({ book }: Props) {
     const bestClass = book.bestInCategory ? ' best' : '';
+    const [active, setActive] = React.useState(false);
 
-    const className = classNames({
+    const booksClassName = classNames({
         book: true,
         best: book.bestInCategory,
         'mb-3': true,
     });
 
+    const bookCoverClassName = classNames('book-cover', active && 'active');
+
+    const prefaceClassName = classNames(
+        'preface bg-body-tertiary',
+        active && 'active'
+    );
+
+    const toogleBook = () => {
+        setActive(!active);
+    };
```

Comment:
Move this to after `useState` so related code stay close.
```suggestion
    const toogleBook = () => setActive(!active);
```

Reply:
done, Thanks, it more cleaner now!